bedrock/products/templates/products/vpn/more/why-mozilla-vpn.html

```diff
@@ -0,0 +1,69 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "products/vpn/more/base-article.html" %}
+
+{% block page_css %}
+  {{ css_bundle('mozilla-vpn-info') }}
+{% endblock %}
+
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
+
+{% block page_title %} Why Mozilla VPN? {% endblock page_title %}
+{% block page_desc %}
+Learn why you should use a VPN made by Mozilla - a company that has a track record of putting people first and fighting
+for online privacy.
+{% endblock page_desc %}
+
+{% set source = '?source=vpn-info-why-mozilla' %}
+
+{% block article %}
+
+<h3>Privacy should not be optional</h3>
+<p>Hiding your identity and cloaking your web traffic can seem like something only super tech-savvy people do. But privacy is not something that you should treat as optional—it requires vigilance and protection to keep hold of it. Mozilla VPN offers you security, reliability and speed on every device, wherever you are—while supporting privacy and security and freedom online.</p>
+<p><strong><a href="{{ url('mozorg.about.manifesto') }}#principles">Mozilla Principle #4:</a> Individuals’ security and privacy on the internet are fundamental and must not be treated as optional.</strong></p>
+
+<h3 class="c-section-heading">Here's why Mozilla VPN is the best VPN:</h3>
```

Comment:
Nit: should be a curly apostrophe (it's only really noticeable because the font is so big)
```suggestion
<h3 class="c-section-heading">Here’s why Mozilla VPN is the best VPN:</h3>
```